core/modules/contact/contact.install

```diff
@@ -28,6 +28,46 @@ function contact_install() {
   }
 }
 
+/**
+ * Implements hook_enable().
+ */
+function contact_enable() {
+  // Build menu router before creating a menu item to the /contact path.
+  menu_router_build();
+
+  // Get main-menu max weight
+  $query = db_select('menu_links')
+    ->condition('menu_name' , 'main-menu');
+  $query->addExpression('MAX(weight)');
+  $weight = $query->execute()->fetchField();
+
+  // Create a Contact link in the main menu.
+  $item = array(
+    'link_title' => t('Contact'),
+    'link_path' => 'contact',
+    'menu_name' => 'main-menu',
+    'weight' => $weight + 1, // Last menu item.
+  );
+  menu_link_save($item);
```

Comment:
Add a message (backdrop_set_message) to inform the admin user that a link have been created ?

Reply:
I would say that best to do so, yes.

Reply:
@klonos any help for the wording ? ;)